releases/latest/beta/helpers/build/populate_scc.sh

```diff
@@ -110,7 +110,7 @@ done
 # restore umask
 umask ${OLD_UMASK}
 
-rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache && chmod -R g+rwx /output/workarea
+rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache /output/workarea
```

Comment:
@ymanton comment on line 104 states "Server start/stop to populate the /output/workarea and make subsequent server starts faster.", so wondering if removing /output/workarea would affect server startup time? 

@idlewis Some comparison runs of sample apps with and without this change (like in this [PR](https://github.com/OpenLiberty/ci.docker/pull/228#issue-527759179)) would help to be certain.

Reply:
The comment predates my involvement so I don't know exactly what it's referring to, but I already confirmed with @idlewis that removing this directory shouldn't affect the SCC.